analysis/tests/src/Diagnostics.res

```diff
@@ -0,0 +1,3 @@
+let = 1 + 1.0
+let add = =2
```

Comment:
Currently it's trying to compile this file.
Notice you don't need to compile the file in order to run your test.
Only, the testing infra is set up to always compile.
Looks like you'd want to split so that some non-compiled files can be tested too.

Reply:
So e.g. `tests/not_compiled` could contain those tests, or some other dir name.